packages/qcow-format/qcow-format.0.4.2/opam

```diff
@@ -21,7 +21,7 @@ depends: [
   "mirage-types-lwt" {< "3.0.0"}
   "lwt"
   "mirage-block" {>= "0.2"}
-  "mirage-block-unix" {>= "2.1.0"}
+  "mirage-block-unix" {>= "2.1.0" & < "2.5.0"}
```

Comment:
It's a bit tight since `mirage-block-unix.2.5.0` implements the Mirage v3 APIs. Perhaps we should bump the major version to `mirage-block-unix.3.0.0`?

Looking at this patch again, I don't understand why `mirage-types-lwt {< "3.0.0" }` wasn't enough. Unfortunately my build log has expired. Perhaps I'll close this for now and rethink.